flexget/components/series/next_series_episodes.py

```diff
@@ -55,6 +55,8 @@ def search_entry(self, series, season, episode, task, rerun=True):
         paren_match = re.match(r'(.+?)( \(.+\))?$', series.name)
         if paren_match.group(2):
             alts.append(paren_match.group(1))
+        if '...' in series.name and series.name.replace('...') not in alts:
+            alts.append(series.name.replace('...'))
```

Comment:
Looks like you are missing the blank argument here in these replace statements. i.e. `...replace('...', '')`

Reply:
Omg, sorry! Fixed now :)